pywinauto/controls/common_controls.py

```diff
@@ -3388,6 +3388,58 @@ def set_view(self, viewType):
         if res == 0:
             raise RuntimeError('Failed to set view in Calendar')
 
+    #----------------------------------------------------------------
+    def set_day_states(self, month_states):
+        """Sets the day states for all months that are currently visible within a month calendar control"""
+        remote_mem = RemoteMemoryBlock(self)
+        day_states = (ctypes.c_uint32 * len(month_states))(*month_states)

```

Comment:
I see `typedef DWORD MONTHDAYSTATE` in [the MSDN docs](https://msdn.microsoft.com/en-us/library/windows/desktop/bb760915(v=vs.85).aspx). `DWORD` is defintely smaller than `ctypes.c_uint32`. You may check it by `ctypes.sizeof()` function. How about using `from ctypes import wintypes; wintypes.DWORD` instead?